treeherder/perf/models.py

```diff
@@ -134,6 +134,23 @@ class PerformanceAlert(models.Model):
                                         null=True)
     series_signature = models.ForeignKey(PerformanceSignature)
     is_regression = models.BooleanField()
+
+    UNTRIAGED = 0
+    INVALID = 1
+    WONTFIX = 2
+    INVESTIGATING = 3
+    RESOLVED = 4
+    DUPLICATE = 5
+
+    STATUSES = ((UNTRIAGED, 'Untriaged'),
+                (INVALID, 'Invalid'),
+                (WONTFIX, 'Won\'t fix'),
```

Comment:
bugzilla solves this with WONTFIX.  I am fine with proper english- just stood out as I was reading the code.